internal/config/config.go

```diff
@@ -397,15 +405,15 @@ func ExportersFromViperConfig(logger *zap.Logger, v *viper.Viper) ([]exporter.Tr
 		name string
 		fn   func(*viper.Viper) ([]exporter.TraceExporter, []exporter.MetricsExporter, []func() error, error)
 	}{
-		{name: "datadog", fn: exporterparser.DatadogTraceExportersFromViper},
-		{name: "stackdriver", fn: exporterparser.StackdriverTraceExportersFromViper},
-		{name: "zipkin", fn: exporterparser.ZipkinExportersFromViper},
-		{name: "jaeger", fn: exporterparser.JaegerExportersFromViper},
-		{name: "kafka", fn: exporterparser.KafkaExportersFromViper},
-		{name: "opencensus", fn: exporterparser.OpenCensusTraceExportersFromViper},
-		{name: "prometheus", fn: exporterparser.PrometheusExportersFromViper},
-		{name: "aws-xray", fn: exporterparser.AWSXRayTraceExportersFromViper},
-		{name: "honeycomb", fn: exporterparser.HoneycombTraceExportersFromViper},
+		{name: "datadog", fn: datadogexporter.DatadogTraceExportersFromViper},
```

Comment:
Now that these are separate packages, we should rename the functions to just `FromViper(...)` perhaps, so we don't have the "stutter" that people sometime talk about.

Reply:
I second @sjkaris' suggestion!

Reply:
I don't know how far you want to go but there are a series of similar good renames after they go to their own packages, e.g.: zipkinexporter.ZipkinConfig -> zipkinexporter.Config ...

Reply:
@pjanotti happy to review some PRs :+1: 

I would play with the code more, but I think this is a good start.

Reply:
It is a great start! I will try to help with some of those too.